src/cli/project/version/bump.rs

```diff
@@ -0,0 +1,36 @@
+use crate::Project;
+use rattler_conda_types::VersionBumpType;
+
+pub async fn execute(mut project: Project, bump_type: VersionBumpType) -> miette::Result<()> {
+    // get version and exit with error if not found
+    let current_version = project
+        .version()
+        .as_ref()
+        .ok_or_else(|| {
+            eprintln!(
+                "{}No version found in manifest.",
+                console::style(console::Emoji("✖ ", "")).red(),
+            );
+        })
+        .unwrap()
+        .clone(); // NOTE(hadim): ideas are welcome to get rid of that clone.
+
+    // bump version
+    let new_version = current_version.bump(bump_type).unwrap();
```

Comment:
Please dont unwrap. Instead return a descriptive error for the user.

Reply:
Used `.map_err(|_| miette::miette!("Unable to bump version."))?;`

Reply:
I think .into_diagnostic().context("unable to bump version") might be better. That way you preserve the source error which will be more informative to the user.